modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/tool/java/parser/ResourceOpenAPIParser.java

```diff
@@ -1168,13 +1194,23 @@ private static boolean _isSchemaMethod(
 	private static boolean _isValidParameter(String name, String schemaName) {
 		String schemaVarName = StringUtil.lowerCaseFirstLetter(schemaName);
 
-		if (!name.equals("id") && !name.equals(schemaVarName + "Id") &&
-			!name.equals(schemaVarName)) {
+		if (StringUtil.equals(name, "aggregation") ||
+			StringUtil.equals(name, "aggregationTerms") ||
+			StringUtil.equals(name, "fields") ||
```

Comment:
@ccorreagg `fields` is valid parameter. See com.liferay.headless.batch.engine.internal.resource.v1_0.ExportTaskResourceImpl#postExportTask

also see example of generated BaseAccountResourceImpl it passes `null` instead of fields parameter:
https://github.com/liferay-headless/liferay-portal/pull/1057/commits/325239e45999c7792457122b7c798aefe45f7da0#diff-76cb137421f9d51b3aba87cce4f05a69806b2dcca9ed269a07fd33e80950ce40R275

Reply:
`fields` parameter if set tells which fields would be exported. If not set all fields would be exported

Reply:
@ccorreagg now I see OpenAPI fields to be included in export file are passed in:
`fieldNames` query parameter.
Ignore my comment about `fields`
Instead of null we can extract `fieldNames` from `UriInfo` object